### PR TITLE
ADCM-3076 Refactor rbac/tests from pytest to unittest, rework its structure

### PR DESCRIPTION
**Pull request by SofiaTal**

### CODE REVIEW

File: python/rbac/tests/test_group.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                self.assertEqual(
+                    actual_value,
+                    expected_value,
+                    f"{g}: wrong {attr} (`{actual_value}`," f" expected: `{expected_value}`)",
+                )
+
+            g.delete()
+
+            try:
+                Group.objects.get(pk=g_pk)
```
[the remark this answers is not in the document]
Comment (SofiaTal) [reply python/rbac/tests/test_group.py]: Done!

File: python/rbac/tests/test_policy.py [hunk 2]
```diff
@@ -0,0 +1,513 @@
...
+    Host,
+    HostProvider,
+    Prototype,
+    ServiceComponent,
+)
+from rbac.models import Group, Policy, User
+from rbac.tests.test_base import BaseTestCase
+
+
+class PolicyTest(BaseTestCase):  # pylint: disable=too-many-instance-attributes
```
[the remark this answers is not in the document]
Comment (SofiaTal) [reply python/rbac/tests/test_policy.py:26]: Refactored in all files, too many instance attribute is actual disable

File: python/rbac/tests/test_role.py [hunk 3]
```diff
@@ -0,0 +1,553 @@
...
+        r = Role(module_name="rbac", class_name="qwe")
+        with self.assertRaises(AdwpEx) as context:
+            r.get_role_obj()
+            self.assertEqual(context.exception.error_code, "ROLE_CLASS_ERROR")
+
+        r = Role(module_name="rbac.roles", class_name="ModelRole")
+        obj = r.get_role_obj()
+        self.assertTrue(isinstance(obj, ModelRole))
+
+    # pylint: disable=protected-access
```
[the remark this answers is not in the document]
Comment (SofiaTal) [reply python/rbac/tests/test_role.py]: Moved

File: python/rbac/tests/test_role.py [hunk 4]
```diff
@@ -0,0 +1,553 @@
...
+                script="./action.yaml",
+                script_type="ansible",
+                state_available="any",
+                prototype=self.cop_22,
+                display_name="Component 2 from Service 2 Action",
+            ),
+        ]
+        Action.objects.bulk_create(actions)
+
+    def test_cook_roles(self):  # pylint: disable=redefined-outer-name
```
[the remark this answers is not in the document]
Comment (SofiaTal) [reply python/rbac/tests/test_role.py]: Old pylint disable

**Review by winmasta — APPROVED** [review]

**Comment by SofiaTal** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @SofiaTal! Build has been launched!

**Pull request merged**